src/site/antora/modules/ROOT/pages/faq.adoc

```diff
@@ -18,7 +18,7 @@ Licensed to the Apache Software Foundation (ASF) under one or more
 = F.A.Q.
 
 This page compiles a list of frequently asked questions.
-If you don't find the answer to your question, please consult to link:{logging-services-url}/support.html[the support page].
+If you don't find the answer to your question, please consult the link:{logging-services-url}/support.html[the support page].
```

Comment:
```suggestion
If you don't find the answer to your question, please consult link:{logging-services-url}/support.html[the support page].
```

"The" is repeated otherwise.